app/code/Magento/Developer/Console/Command/TemplateHintsShowCommand.php

```diff
@@ -0,0 +1,66 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\Developer\Console\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Magento\Framework\App\Config\ConfigResource\ConfigInterface;
+
+class TemplateHintsShowCommand extends Command
+{
+
+    /**
+     * command name
+     */
+    const COMMAND_NAME = 'dev:template-hints:show';
+
+    /**
+     * Success message
+     */
+    const SUCCESS_MESSAGE = "Template hints are ";
```

Comment:
Add placeholder there: `Template hints are %status` and then you can use Phrase object: `__(self::SUCCESS_MESSAGE, ['status' => $status])`

Reply:
Done!